src/pages/graphql/schema/attributes/index.md

```diff
@@ -3,3 +3,5 @@ title: Attributes
 ---
 
 # Attributes
+
+The attributes queries allow you to retrieve values stored as custom and [Entity-Attribute-Value (EAV)()] attributes. Custom attributes are those added on behalf of a merchant. For example, a merchant might need to add attributes to describe products, such as shape or volume. EAV attributes are defined by the system.
```

Comment:
Should the first occurrence of EAV link somewhere? The formatting looks off; like it's meant to use a link definition, but there is no definition in this file.